glob/braces.in

```diff
@@ -6,6 +6,10 @@ root=true
 [*.{py,js,html}]
 choice=true
 
+; word choice with spaces
+[*.{py, js, html}]
```

Comment:
```suggestion
[*.{pl, txt, htm}]
```
If the extensions are different from those in the previous block, it will be easier to find the test case while fixing a bug.